mangle.json

```diff
@@ -60,8 +60,8 @@
       "$_processingException": "__",
       "$_globalContext": "__n",
       "$_context": "c",
-      "$_defaultValue": "__",
```

Comment:
One potential issue here is https://github.com/preactjs/prefresh/blob/main/packages/babel/src/index.mjs#L404 where `_defaultValue` might fail to update as it's not on `__` anymore. Instead we'd update the `_parent` of `Context` which sounds... problematic